app/controllers/api/v0/listings_controller.rb

```diff
@@ -1,48 +1,46 @@
 module Api
   module V0
-    class ClassifiedListingsController < ApiController
+    class ListingsController < ApiController
       include Pundit
-      include ClassifiedListingsToolkit
+      include ListingsToolkit
 
       before_action :authenticate_with_api_key_or_current_user!, only: %i[create update]
       before_action :authenticate_with_api_key_or_current_user, only: %i[show]
 
-      before_action :set_classified_listing, only: %i[update]
+      before_action :set_listing, only: %i[update]
 
       before_action :set_cache_control_headers, only: %i[index show]
 
       skip_before_action :verify_authenticity_token, only: %i[create update]
 
       def index
-        @classified_listings = ClassifiedListing.published.
+        @listings = Listing.published.
           select(ATTRIBUTES_FOR_SERIALIZATION).
-          includes(:user, :organization, :taggings, :classified_listing_category)
+          includes(:user, :organization, :taggings, :listing_category)
 
         if params[:category].present?
-          category = ClassifiedListingCategory.find_by(slug: params[:category])
-          @classified_listings =
-            @classified_listings.where(classified_listing_category: category)
+          @listings = @listings.in_category(params[:category])
         end
-        @classified_listings = @classified_listings.order(bumped_at: :desc)
+        @listings = @listings.order(bumped_at: :desc)
 
         per_page = (params[:per_page] || 30).to_i
         num = [per_page, 100].min
         page = params[:page] || 1
-        @classified_listings = @classified_listings.page(page).per(num)
+        @listings = @listings.page(page).per(num)
 
-        set_surrogate_key_header ClassifiedListing.table_key, @classified_listings.map(&:record_key)
+        set_surrogate_key_header Listing.table_key, @listings.map(&:record_key)
```

Comment:
These keys will actually stay the same since the table name is the same. I think that's fine.